src/bootstrap/react.bootstrap.js

```diff
@@ -27,25 +31,53 @@ module.exports = function(dependentModulesMap) {
     elem.textContent = '';  // Clear the existing content
 
     // Babel is defined in the React live-update template via an external script
-    var result = Babel.transform('_secretReactRenderMethod(' + newContent + ', elem);', { presets: ['es2015', 'react'] });
+    var result;
+
+    try {
+      result = Babel.transform('_secretReactRenderMethod(' + newContent + ', elem);', { presets: ['es2015', 'react'] });
+    } catch (err) {
+      console.log(err);
+    }
 
     // Strangely, Babel doesn't eval the code automatically
-    eval(result.code);
+    if (result) {
+      eval(result.code);
+    }
   }
 
 
-  function onChangeLiveEditReact(event, targetCSSClassName, doc) {
-    var elem = doc.querySelector(targetCSSClassName);
-
-    reactCompile(elem, event.srcElement.value);
+  function onChangeLiveEditReact(element, value) {
```

Comment:
Originally this function was exported so that a host project could adapt the behaviour somewhat (eg. Denounce the change events, ...?). Now that CodeMirror is being used, maybe this isn't necessary (nor is the debounce() needed?).